### ISSUE TITLE
Fix reverse ens

### ISSUE THREAD

**Issue by milkyklim** [iearn-finance/iearn-finance#155]
This one is merged but doesn't work right now: https://github.com/iearn-finance/iearn-finance/pull/86.

**Comment by x48-crypto** [comment]
```
commit cba824a98eae66d4a25064db612b4b71e704329e
Author: x48 <x48114@protonmail.com>
Date:   Sun Oct 25 20:04:01 2020 -0500

    fix: fix reverse ENS regression (#155)
```

**Comment by x48-crypto** [comment]
This was broken by #118
